code/extensions/SolrSearch.php

```diff
@@ -393,8 +393,13 @@ public function getQuery() {
 			if (!$sortBy) {
 				$sortBy = 'score';
 			}
-			
-			$sortDir = $sortDir == 'Ascending' ? 'asc' : 'desc';
+
+			if ($sortDir == 'Ascending') {
+				$sortDir = 'ASC';
+			}
+			else if ($sortDir == 'Descending') {
+				$sortDir = 'DESC';
+			}
 
 			$builder->sortBy($sortBy, $sortDir);
```

Comment:
This is a good point. I'm pretty sure the search engine just errors and returns nothing, but I shouldn't make that assumption and rely on it. I will fix this up.